css/properties/overflow-anchor.json

```diff
@@ -18,7 +18,7 @@
               "version_added": "66"
             },
             "firefox_android": {
-              "version_added": false
```

Comment:
false is correct here: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/66#CSS

Reply:
Fixed!